### PR TITLE
PR for Capstone project for Microverse 1st module

### PR DESCRIPTION
**Pull request by DarioAlessioR**
This PR has been opened to ask for approval of branch capstone1 to be merged into the master branch.

The project is part of the activity "HTML + basic JavaScript Capstone" of week 5 module 1 of the Microverse program.

The project has been completed according to the requirements established for this activity, which include the following:

- You should personalize the content of your page. Choose a topic that is different than the one in the original design.

- You should follow these design guidelines, including:
  Colors.
  Typography: font face, size and weight.
  Layout: composition and space between elements.
  The pages should look almost identical to the original design. Small adjustments like text or image changes are 
  acceptable.

- You can use a CSS framework (for example Bootstrap) for styling, if you want to.

- You should build only these 2 pages:

  The home page.

  The about page.

Each of these pages should have versions for 2 different screen sizes:

  Mobile: up to 768px wide.

  Desktop: 768px or wider.

Interactions

- Links
The home page should have a link in the menu to the about page.
The logo in the header links to the home page.
Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
The mobile menu should have a close (X) button that closes the menu.
Dynamic page

- The section "Featured speakers" should be created dynamically in JavaScript.

- You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

The project has been done following the template provided by Microverse in the following link:
https://www.behance.net/gallery/29845175/CC-Global-Summit-2015

This PR includes also a 5 minutes presentation video of the project that can be watched by clicking this link: 
https://www.loom.com/share/4dfe164e43f549a59974d6e1055f83a4

Thanks in advance for reviewing it and for your improvement suggestions.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+            <h4>See past New Bands Rock Festival Wall of Sound editions</h4>
+
+            <p>Take a look at the past 2 editinos of our main event celebrated in Houston and Los Angeles in 2018 and 2019.  
+            </p>
+
+            <div class="past-editions">
+              <img src="images/concert4.png" title="New Bands Festival 2018" alt="New Bands Festival 2018">
+              <img src="images/concert5.png" title="New Bands Festival 2019" alt="New Bands Festival 2019">
+            </div>
```
Comment (AnselemOdims) [inline about.html]: - [ ] Please ensure that this matches the design. The past concerts cards should be stacked on top of each other.
- [ ] Also, it is advisable use proper background images added using the `background-image` CSS property.

### Your Design
image



### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @DarioAlessioR , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Really wonderful work you have done here, but you were supposed to include one thing you are most proud about in the project.
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by DarioAlessioR** [comment]
Atention: Mr. @AnselemOdims .

All required changes are done on the project:
- About images are now background images.
- Images stack one over another on the mobile version.
- Images aspect is now closer to the template aspect, although not identical due to personalization.

In addition to these changes, a new video is attached to fulfill the requirement of mentioning one part of the project that makes the author feel proud.
https://www.loom.com/share/2ef0caaff0fa439da7e7fff618fbecf3

Thank you in advance for your review.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @DarioAlessioR  😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done addressing the required changes from the previous review. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**